docs/node-mixin/dashboards/network.libsonnet

```diff
@@ -0,0 +1,747 @@
+        legendFormat='CPU {{cpu }} dropped',
+      ))
+      .withDecimals(1)
+      .withUnits('pps')
+      .withNegativeYByRegex('dropped')
+      .withAxisLabel('Dropped(-) | Processed(+)'),
+
+    local networkSoftnetSqueezePanel =
+      nodeTimeseries.new('Softnet Out of Quota')
+      .addTarget(commonPromTarget(
+        'irate(node_softnet_times_squeezed_total{%(nodeQuerySelector)s}[$__rate_interval])' % config { nodeQuerySelector: c.nodeQuerySelector },
+        legendFormat='CPU {{cpu}} out of quota',
+      ))
+      .withDecimals(1)
+      .withUnits('pps'),
+
+    local networkInterfacesTable =
+      nodePanels.table.new(
+        title='Network Interfaces Overview'
+      )
+      // "Value #A"
+      .addTarget(commonPromTarget(
+        expr='node_network_up{%(nodeQuerySelector)s}' % config { nodeQuerySelector: c.nodeQuerySelector },
+        format='table',
+        instant=true,
+      ))
+      // "Value #B"
+      .addTarget(commonPromTarget(
+        expr='node_network_carrier{%(nodeQuerySelector)s}' % config { nodeQuerySelector: c.nodeQuerySelector },
+        format='table',
+        instant=true,
+      ))
+      // "Value #C"
+      .addTarget(commonPromTarget(
+        expr=q.networkTransmitBitsPerSec,
+        format='table',
+        instant=true,
+      ))
+      // "Value #D"
+      .addTarget(commonPromTarget(
+        expr=q.networkReceiveBitsPerSec,
+        format='table',
+        instant=true,
+      ))
+      // "Value #E"
+      .addTarget(commonPromTarget(
+        expr='node_arp_entries{%(nodeQuerySelector)s}' % config { nodeQuerySelector: c.nodeQuerySelector },
+        format='table',
+        instant=true,
+      ))
+      // "Value #F"
+      .addTarget(commonPromTarget(
+        expr='node_network_mtu_bytes{%(nodeQuerySelector)s}' % config { nodeQuerySelector: c.nodeQuerySelector },
+        format='table',
+        instant=true,
+      ))
+      // "Value #G"
+      .addTarget(commonPromTarget(
+        expr='node_network_speed_bytes{%(nodeQuerySelector)s} * 8' % config { nodeQuerySelector: c.nodeQuerySelector },
+        format='table',
+        instant=true,
+      ))
+      // "Value #H"
+      .addTarget(commonPromTarget(
+        expr='node_network_transmit_queue_length{%(nodeQuerySelector)s}' % config { nodeQuerySelector: c.nodeQuerySelector },
+        format='table',
+        instant=true,
+      ))
+      // "VALUE #I"
+      .addTarget(commonPromTarget(
+        expr='node_network_info{%(nodeQuerySelector)s}' % config { nodeQuerySelector: c.nodeQuerySelector },
+        format='table',
+        instant=true,
+      ))
+      // "VALUE #J"
+      // .addTarget(commonPromTarget(
+      //   expr='node_network_protocol_type{%(nodeQuerySelector)s}' % config {nodeQuerySelector: c.nodeQuerySelector},
+      //   format="table",
+      //   instant=true,
+      // ))
```

Comment:
Is this needed?

Or the `.merge()` which is commented out below?
```suggestion
```

Reply:
removed